src/services/decoders/ClpIrDecoder/index.ts

```
@@ -26,6 +26,9 @@
     getStructuredIrNamespaceKeys,
     StructuredIrNamespaceKeys,
 } from "./utils";
+import {
+    parseFilterKeys,
+} from "../utils";
```

ESLint

[error] Imports must not be broken into multiple lines if there are 1 or less elements.

(import-newlines/enforce)